docusaurus/docs/guides/deployments/10-linux/10-controller/10-deploy.mdx

```diff
@@ -0,0 +1,98 @@
+
+Disable bootstrapping a PKI by setting `ZITI_BOOTSTRAP_PKI=false` in `/opt/openziti/etc/controller/service.env`.
+
+Check out [the PKI page](/learn/core-concepts/pki.md) for an overview of these concepts.
+
+### Generating a Configuration
+
+The controller service will generate a configuration during the first startup.
+
+The controller's configuration is loaded from a YAML file ([reference](/reference/30-configuration/controller.md)). The Linux system service will generate a valid configuration during the first startup unless one already exists. 
+
+The filename of the configuration file, relative to the Linux service's working directory (`/var/lib/ziti-controller`), is specified in `/lib/systemd/system/ziti-controller.service` as an argument to the `ExecStart` command. You may override this in `/etc/systemd/system/ziti-controller.service.d/override.conf`.
+
+Disable bootstrapping a configuration by setting `ZITI_BOOTSTRAP_CONFIG=false` in `/opt/openziti/etc/controller/service.env`.
+
+### Bootstrapping the Database
+
+The controller service will bootstrap a database during the first startup.
+
+The controller requires a BoltDB database to store its state. The Linux system service will initialize a database with a default admin password during the first startup unless one already exists.
+
+You must specify the management password for the default admin user before starting the service. This is done by setting `ZITI_PWD` in `/opt/openziti/etc/controller/bootstrap.env` or in the root-only file specified by `LoadCredential` in `/etc/systemd/system/ziti-controller.service.d/override.conf`. You may delete the password after bootstrapping for security.
+
+Disable bootstrapping the database by setting `ZITI_BOOTSTRAP_DATABASE=false` in `/opt/openziti/etc/controller/service.env`.
+
+## Firewall
+
+The controller listens on a single configurable TCP port: `1280/tcp`. This TLS server employs SNI to select the correct certificate for presentation when there are multiple certificates. Ziti clients use ALPN to negotiate a connection to the router control plane (`ziti-ctrl`) or the REST APIs (`h2`, `http/1.1`).
+
+You may set `ZITI_CTRL_ADVERTISED_PORT` in `/opt/openziti/etc/controller/bootstrap.env` to bootstrap with a different port.
+
+Clients "learn" the controller's address and port when they enroll, so it is necessary to re-enroll or replace the identity if the controller's advertised address or port changes.
+
+## Agent
+
+The controller provides an IPC agent for administration. The agent listens on a Unix domain socket inside the filesystem namespace of the controller service. Here's an example for querying the controller's agent for statistics.
+
+```text
+systemctl show -p MainPID --value ziti-controller.service \
+| xargs -rIPID sudo nsenter --target PID --mount -- \
+    ziti agent stats
+```
+
+```buttonless title="Output"
+goroutines: 50
+OS threads: 22
+GOMAXPROCS: 16
+num CPU: 16
+```
+
+## Logging
+
+<CliLogFormats/>

```

Comment:
Mentioned prior, but this section feels like it needs to be "you will log with json, you will use journald, you will ship your logs" type of vibe. Also the included logging refers to ziti-edge-tunnel and just doesn't feel reusable